### ISSUE TITLE
URLs used in the example cause serious issues with Google safety check

### ISSUE THREAD

**Issue by rustamli** [passport/express-4.x-google-oauth2-example#1]
Mentioning `accounts.google.com` in the urls like `/oauth2/redirect/accounts.google.com` triggers Google security checks and website gets marked as a potential phishing attack source. It is very bad for the domain as it can disappear from Google search results until it is resolved. Changing paths so that they don't mention "google.com" fixes this.

**Comment by jaredhanson** [comment]
Interesting.  I was not aware of this.  Is there any official documentation from Google regarding this, or forum discussions from people who have experienced this?  I'd like to document the rationale along with changes.

**Comment by rustamli** [comment]
@jaredhanson thanks for a quick response. Documentation on this doesn't make sense as it only helps phishing attackers camouflage their attacks better. I am also not sure if this is triggered on Chrome level or on Google Auth process level. 

I have experienced this with two different domains now. And this is what I get on Google Search Console:


> 1 issue detected. Google has detected harmful content on some of your site’s pages. We recommend that you remove it as soon as possible. Until then, browsers such as Google Chrome will display a warning when users visit or download certain files from your site.

And I am referred to this [link](https://support.google.com/webmasters/answer/9044101#phising) for more details.

The website is nothing but deployment of this particular repo with appropriate Client ID and Secret configuration.

Luckily, I was playing with a non-important domain for a side-project and nothing major is affected. But it's been days and Google still hasn't switched off security warnings for my website, even though I have submitted a request with explanation of what happened.
